app/models/user.rb

```diff
@@ -78,6 +78,7 @@ def four_different_characters
 
   has_many :team_users
   has_many :teams, through: :team_users, source: :team
+  has_many :real_teams, -> { where single: false }, through: :team_users, source: :team
```

Comment:
maybe naming could be more descriptive than "real" as in "non-single-teams" or such...

Reply:
Good idea—I was not very happy with the name either. I'm gonna change it right now. 🙂

Reply:
Resolved in 329929b